src/js/core.js

```diff
@@ -1088,6 +1138,66 @@
                 target.innerHTML = html;
                 this.events.updateInput(target, { target: target, currentTarget: target });
             }
+        },
+
+        addElements: function (elements) {
```

Comment:
Yes i've checked that again.. i was implementing my addElements method accordingly to the things i did find in "setup".. so i closely looked into `createElementsArray` and i thought, the things with the selector and quering the elements should not be "executed again" - it feels natural to me that i call an "addElements" method already with the elements as an array, not just giving him a new selector-string.

So, after looking closely into it again with the `<textarea>` in mind, the only code that actually should re-run is this part of `createElementsArray`:

`his.elements = [];`
`        elements.forEach(function (element, index) {`
`            if (element.nodeName.toLowerCase() === 'textarea') {`
`                this.elements.push(createContentEditable.call(this, element, index));`
`            } else {`
`                this.elements.push(element);`
`            }`
`        }, this);`

It is only the last loop which checks again all elements of his array, and if he finds anything as <textarea> - he converts it into a div.

So, what about if we extract this little loop into a new local function, maybe "_convertTextareas" and, we can use it at both parts? in `createElementsArray` and in my `addElements` ;) It is the only thing i see missing from the parts i let re-run i would say..